rpcd/playbooks/roles/rpc_maas/templates/network_throughput.yaml.j2

```diff
@@ -1,4 +1,4 @@
-{% set max_speed = (item.0.max_speed | default(0, boolean=true) or item.1.stdout) | int * 10 ** 6 / 8 %}
+{% set max_speed = (item.0.max_speed | default(item.1.stdout | default(1000, boolean = true), boolean = true)) | int * 10 ** 6 / 8 %}
```

Comment:
They don't have to set it, autodetect is still the default, all I've done is add sensible default for when autodetect fails. IMO 0 is not a sensible default as it will usually cause alarms.